### DISCUSSION TITLE
Informações da porta perdidas (Python, Flask, Docker)

### DISCUSSION THREAD

**Discussion by yooung1** [frontendbr/forum#2264] [Perguntas e Respostas] [answered]
estou com um pequeno problema, criei um container no Docker usando a porta 8080, quando entro no navegador e digito

Localhost:8080 

O meu container da erro

"Não é possível acessar esse site" 

Quando início o programa pelo Python usando o framework Flask, o programa funciona, mas a partir do momento em que insiro ele dentro do container ele não é visualizado na porta selecionada. Alguém poderia me ajudar com isto por favor? :)

**Comment by igorlvicente** [comment, marked as answer]
Acho que isso tem a ver com o Docker. Você está "ligando" (não sei o termo correto) a porta dentro do container com a porta fora dele? Não lembro bem como faz, mas é algo tipo `8080:8080` em algum lugar do comando ou do Dockerfile.
